src/handlers/notification/index.js

```diff
@@ -446,6 +446,56 @@ const processMessage = async (msg, span) => {
     return true
   }
 
+  // special event emitted by central-ledger when the Payee sent a status of `RESERVED` in PUT /transfers/{ID}
+  // and the ledger failed to commit the transfer
+  if (actionLower === ENUM.Events.Event.Action.RESERVED_ABORTED) {
+    if (Config.PROTOCOL_VERSIONS.CONTENT !== '1.1') {
+      Logger.isDebugEnabled && Logger.debug(`Notification::processMessage - Action: ${actionLower} - Skipping reserved_aborted notification callback (${from}).`)
+      return
+    }
+
+    const callbackURLTo = await Participant.getEndpoint(to, ENUM.EndPoints.FspEndpointTypes.FSPIOP_CALLBACK_URL_TRANSFER_PUT, id, span)
```

Comment:
Looking at the endpoint here, Miguel do you think we need to add a new one for Patch in the seeds? similar to the one for put? https://github.com/mojaloop/central-ledger/blob/fad85b03128908133d72255f1251c04b40342b01/seeds/endpointType.js#L41